aws/table_aws_lambda_function.go

```diff
@@ -362,6 +362,17 @@ func getLambdaFunctionUrlConfig(ctx context.Context, d *plugin.QueryData, h *plu
 
 	functionName := functionName(h.Item)
 
+	commonColumnData, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+
+	awsCommonData := commonColumnData.(*awsCommonColumnData)
+	if awsCommonData.Partition == "aws-us-gov"{
```

Comment:
```suggestion
	if awsCommonData.Partition == "aws-us-gov" {
```

@niharika2k00 can you please either run `gofmt` on your code to format it, or ensure that it's setup in your VS Code so it automatically runs?